### PR TITLE
[Quant] Remove warnings from using torch.tensor(value)

### PR DESCRIPTION
**Pull request by jcaip**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #84277

Summary:

I think zafar made an earlier pull for these changes [here](https://github.com/pytorch/pytorch/commit/ce0786add26c1e117b16b58e8ae12dbe776133e1), but they didn't seem to make it through the migration.

Test Plan:
```
python test/test_quantization.py
```

Reviewers:

Subscribers:

Tasks: https://github.com/pytorch/pytorch/issues/73566

Tags: quant

Differential Revision: [D39145070](https://our.internmc.facebook.com/intern/diff/D39145070)

### CODE REVIEW

**Comment by jcaip** [comment]
@jcaip has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39137663).

**Review by z-a-f — APPROVED** [review]

**Comment by malfet** [comment]
Merge is failing because this PR has already been committed to trunk as https://github.com/pytorch/pytorch/commit/eabe34cc40aeb79a10208df291b2a4d92302fbc2

**Comment by malfet** [comment]
@jcaip just curious, why did you reopened the PR? Just to add comments?

**Comment by jcaip** [comment]
I thought I had to in order to address the githubactions bot and add labels.
